sections/appendix/stemming.tex

```diff
@@ -0,0 +1,40 @@
+\subsection{Stemming the dataset}\label{sec:stemming}
+In this project we have done minimal preprocessing of the dataset, because in a previous project, a more aggressive preprocessing step turned out to have a negative effect on the \gls{lda} model.
```

Comment:
```suggestion
In this project, we have done minimal preprocessing of the dataset, because, in a previous project, a more aggressive preprocessing step turned out to hurt the performance of the \gls{lda} model.
```